### PR TITLE
Views

### PR DESCRIPTION
**Pull request by aeh1707**
In the following post I have created a standard layout along with views.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,19 @@
...
+    <h1><%= @user.name %></h1>
+    <p>Number of posts: <%= @user.posts.count %></p>
+</article>
+<section>
+    <h2>Bio</h2>
+    <p><%= @user.bio %></p>
+</section>
+<section>
+    <h2>Posts</h2>
+    <% @user.posts.first(3).each do |post| %>
```
Comment (Davidosky007) [inline app/views/users/show.html.erb:12]: * Kindly use the `recent_posts` method you declared already in your `user model` to render the user 3 most recent posts as shown in the wireframe 
image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,18 @@
...
+    <article>
+        <h3><%= post.title %></h3>
+        <p><%= post.text.slice(0, 80) + '...' %>
+            <a href="/users/<%= @user.id %>/posts/<%= post.id %>">View this Post</a></p>
+        <section>
+            <p>Comments: <%= post.comments_counter %></p>
+            <p>Likes: <%= post.likes_counter %></p>
+        </section>
+        <section>
+            <% post.comments.each do |comment| %>
```
Comment (Davidosky007) [inline app/views/posts/index.html.erb:11]: * Kindly use the `recent_posts` method you declared already in your `post model` to render the user 5 most recent comments as shown in the wireframe 
* ## WIREFRAME
image
*## YOURS 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @aeh1707 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Good looking readme file 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @aeh1707 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  

H
## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**